packages/tradershub/src/hooks/useMT5AccountHandler.ts

```diff
@@ -0,0 +1,92 @@
+import {
+    useAccountStatus,
+    useActiveTradingAccount,
+    useAvailableMT5Accounts,
+    useCreateMT5Account,
+    useSettings,
+    useTradingPlatformPasswordChange,
+} from '@deriv/api';
+import { MarketType, PlatformDetails } from '../features/cfd/constants';
+import { Jurisdiction } from '../features/cfd/screens/CFDCompareAccounts/constants';
+import { TMarketTypes } from '../types';
+
+type TUseMT5AccountHandler = {
+    marketType: TMarketTypes.All;
+    selectedJurisdiction: string;
```

Comment:
Should we typed the jurisdiction from Api or other source instead of string.
As string would contains anything.

Reply:
How about TMT5LandingCompanyName in types.ts in tradershub ?

Reply:
Dont Use LandingCompanyName as there is going to be a new property `landing_company_name` in `mt5_login_list` which is going to give Broker Name like `Deriv SVG LLC` etc.

Reply:
good catch we have a type for that already `TMT5LandingCompanyName`

Reply:
what api can we get the jurisdiction that wont be changed to something new..?